utils/orion_funcs.py

```diff
@@ -111,10 +111,7 @@ def get_metadata(test,logger):
         dict: dictionary of the metadata
     """
     metadata = {}
```

Comment:
think we can take this line out becuase its being immediately overwritten with the next line

Reply:
Yep, sorry I did not check that! updated in a new push. Thank you!